prog/weave-kube/launch.sh

```diff
@@ -17,8 +21,8 @@ EXPECT_NPC=${EXPECT_NPC:-1}
 
 # Ensure we have the required modules for NPC
 if [ "${EXPECT_NPC}" != "0" ]; then
-    modprobe br_netfilter
-    modprobe xt_set
+    modprobe_safe br_netfilter
+    modprobe_safe xt_set
```

Comment:
Note that this one is actually a module on CentOS.

As far as I see there is an explicit check for br_netfilter further down. Does it also make sense to check for xt_set somehow? Just to not be too forgiving ...